test/unit/config-options/config-options.test.js

```diff
@@ -725,8 +725,6 @@ describe('config', function() {
                 var config = createBrowserConfig({});
                 assert.equal(config.screenshotsDir, '/some/path/gemini/screens');
             });
-
-            //TODO: toplevel
```

Comment:
TODO is a bad practise, and eslint has a rule for that (no-warning-comments). 

For todo should be created a story